src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/PackageVulnerabilityService.cs

```diff
@@ -17,26 +18,26 @@ namespace NuGet.PackageManagement.VisualStudio
     public class PackageVulnerabilityService : IPackageVulnerabilityService
     {
         private readonly IEnumerable<SourceRepository> _sourceRepositories;
-        private GetVulnerabilityInfoResult _vulnerabilities;
+        private readonly AsyncLazy<GetVulnerabilityInfoResult> _vulnerabilitiesLazy;
```

Comment:
Do we require renaming these with "Lazy" in the name? I'd lean against that rename, as a preference.